osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/Hls5Connector.java

```diff
@@ -94,8 +99,12 @@ protected void setSecurity(final DlmsDevice device, final TcpConnectionBuilder t
         final byte[] dlmsAuthenticationKey;
         final byte[] dlmsEncryptionKey;
         try {
-            dlmsAuthenticationKey = this.securityKeyService.getDlmsAuthenticationKey(deviceIdentification);
-            dlmsEncryptionKey = this.securityKeyService.getDlmsGlobalUnicastEncryptionKey(deviceIdentification);
+            Map<SecurityKeyType,byte[]> encryptedKeys = this.securityKeyService.getKeys(deviceIdentification,
+                    Arrays.asList(E_METER_AUTHENTICATION, E_METER_ENCRYPTION));
+            dlmsAuthenticationKey = encryptedKeys.get(E_METER_AUTHENTICATION);
+            dlmsEncryptionKey = encryptedKeys.get(E_METER_ENCRYPTION);
+            //this.securityKeyService.getDlmsAuthenticationKey(deviceIdentification);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)